src/modals/error-modal-content.ts

```diff
@@ -0,0 +1,48 @@
+import {
+  LitElement,
+  html,
+  css,
+  customElement,
+  CSSResult,
+  TemplateResult,
+} from 'lit-element';
+
+/**
+ * This is shown at the bottom of the error modal.
+ *
+ * @export
+ * @class ErrorModalContent
+ * @extends {LitElement}
+ */
+@customElement('donation-form-error-modal-content')
+export class ErrorModalContent extends LitElement {
+  /** @inheritdoc */
+  render(): TemplateResult {
+    return html`
+      <div class="container">
+        <a href="https://help.archive.org/hc/en-us/articles/360037568971-Why-is-there-a-problem-processing-my-donation-" target="_blank">
```

Comment:
Might want to add a `rel="noopener"` for possible performance gains. Since the URL is under our control, you probably don't need to safeguard with "noreferrer".